### PR TITLE
Add improve-design-1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the design is not adjusted for mobile view.
- Currently, the price is not shown on product carts. This is against E-Commerce regulations.
- Currently, the BlackWolf personal coffee form is outdated and not very stable.
- Currently, the wolf is running on a big white border, which is really strange.
- Currently, there is no "Trusted Small" section to display partners.
- Currently, the token handling and authentication is outdated.
- Currently, all images are being loaded seperately by using `<Query>` React-Apollo inline queries.

**What is the new behavior (if this is a feature change)?**
- The design has been improved and the website now looks appealing on most devices.
- The price has been added to the product carts.
- The BlackWolf personal coffee form has been improved and is now very stable.
- The wolf animation now runs on moving grass foreground.
- A new section to display partners has been implemented. (Trusted Small)
- The token handling and authentication has been improved.
- The image urlLinks are now being loaded diretly with `tokenAuth`.
- The CNAME has been changed to fit the new primary website.
- The favicon has been changed.

**Other information**:
- The version is v1.0.0, once this is the first release.

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -1,32 +1,73 @@
...
+    <!-- Primary Meta Tags -->
+    <title>Blue Lupi - DIE KAFFEEALTERNATIVE AUS ÖSTERREICH</title>
+    <meta
+      name="title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
+  AUS ÖSTERREICH"
+    />
+    <meta
+      name="description"
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
```
Comment (pinterid) [inline public/404.html]: The line length limit is 120.

File: public/404.html [hunk 2]
```diff
@@ -1,32 +1,73 @@
...
-			);
-		</script>
-	</head>
-	<body></body>
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.nocoffee.at/" />
+    <meta
+      property="og:title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
```
Comment (pinterid) [inline public/404.html]: Is the newline needed?

File: public/404.html [hunk 3]
```diff
@@ -1,32 +1,73 @@
...
+  AUS ÖSTERREICH"
+    />
+    <meta
+      property="og:description"
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
```
Comment (pinterid) [inline public/404.html]: The max line length is 120.

File: public/404.html [hunk 4]
```diff
@@ -1,32 +1,73 @@
...
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://www.nocoffee.at/" />
+    <meta
+      property="twitter:title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
+  AUS ÖSTERREICH"
+    />
+    <meta
+      property="twitter:description"
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
```
Comment (pinterid) [inline public/404.html]: The max line length is 120.

File: public/index.html [hunk 5]
```diff
@@ -1,141 +1,173 @@
...
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
+    />
+    <meta property="og:image" content="https://www.nocoffee.at/fav/og.jpg" />
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://www.nocoffee.at/" />
+    <meta
+      property="twitter:title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
```
Comment (pinterid) [inline public/index.html]: Is the newline needed?

File: public/index.html [hunk 6]
```diff
@@ -1,141 +1,173 @@
...
+    <meta
+      name="msapplication-TileImage"
+      content="%PUBLIC_URL%/fav/ms-icon-144x144.png"
+    />
+    <meta name="theme-color" content="#ffffff" />
+    <!-- Primary Meta Tags -->
+    <title>Blue Lupi - DIE KAFFEEALTERNATIVE AUS ÖSTERREICH</title>
+    <meta
+      name="title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
```
Comment (pinterid) [inline public/index.html]: Is the new line needed?

File: public/index.html [hunk 7]
```diff
@@ -1,141 +1,173 @@
...
+  AUS ÖSTERREICH"
+    />
+    <meta
+      name="description"
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
```
Comment (pinterid) [inline public/index.html]: Max 120.

File: public/index.html [hunk 8]
```diff
@@ -1,141 +1,173 @@
...
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.nocoffee.at/" />
+    <meta
+      property="og:title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
+  AUS ÖSTERREICH"
+    />
+    <meta
+      property="og:description"
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
```
Comment (pinterid) [inline public/index.html]: Max 120

File: public/index.html [hunk 9]
```diff
@@ -1,141 +1,173 @@
...
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://www.nocoffee.at/" />
+    <meta
+      property="twitter:title"
+      content="Blue Lupi - DIE KAFFEEALTERNATIVE
+  AUS ÖSTERREICH"
+    />
+    <meta
+      property="twitter:description"
+      content="Du hast keine Lust mehr auf herkömmlichen Kaffee? Oder bist du auf der Suche nach einer leckeren, koffeinfreien Alternative?"
```
Comment (pinterid) [inline public/index.html]: Max 120.

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after a few changes.

**Review by pinterid — APPROVED** [review]
Looks good to merge.

File: src/components/helpers/InstagramFeed.jsx [hunk 10]
```diff
@@ -7,61 +7,61 @@ import React from "react";
...
-				`https://www.instagram.com/graphql/query?query_id=17888483320059182&variables={"id":"
+  async componentDidMount() {
+    try {
+      // Hack from https://stackoverflow.com/a/47243409/2217533
+      const response = await fetch(
+        `https://www.instagram.com/graphql/query?query_id=17888483320059182&variables={"id":"
         ${this.props.userId}","first":${this.props.photoCount},"after":null}`
-			);
-			const { data } = await response.json();
+      );
```
Comment (schettn) [inline src/components/helpers/InstagramFeed.jsx:18]: Add newline after multiline var declaration.

File: src/components/helpers/InstagramFeed.jsx [hunk 11]
```diff
@@ -7,61 +7,61 @@ import React from "react";
...
+            src,
+            width,
+            height,
+            url,
+          };
+        }
+      );
+      this.setState({ photos });
+    } catch (error) {
+      //console.error(error);
```
Comment (schettn) [inline src/components/helpers/InstagramFeed.jsx]: Do you really want to ignore the error if it occurs?

File: src/components/organisms/Sections/Features/index.jsx [hunk 12]
```diff
@@ -9,82 +9,57 @@ import { MDBContainer, MDBRow, MDBCol } from "mdbreact";
...
-					></p>
-					<p
-						dangerouslySetInnerHTML={{ __html: reason.value.Column_paragraph }}
-					></p>
-				</MDBCol>
-			);
-		});
+  renderReasons = (data) => {
+    let width =
+      data.whyColumns.length === 4 || data.whyColumns.length === 8 ? "3" : "4";
```
Comment (schettn) [inline src/components/organisms/Sections/Features/index.jsx:15]: Add a newline after a multiline variable declaration.

**Review by schettn — CHANGES_REQUESTED** [review]
Please check.

**Review by schettn — APPROVED** [review]
LGTM! 🐄

File: public/fav/manifest.json [hunk 13]
```diff
@@ -1,41 +1,41 @@
 {
-    "name": "Blue Lupi Homepage",
```
Comment (kleberbaum) [inline public/fav/manifest.json:2]: Why has it been changed to "name": "App"?

File: src/components/organisms/Sections/Features/index.jsx [hunk 14]
```diff
@@ -9,82 +9,58 @@ import { MDBContainer, MDBRow, MDBCol } from "mdbreact";
...
-						className="lead"
-						dangerouslySetInnerHTML={{ __html: reason.value.Column_subhead }}
-					></p>
-					<p
-						dangerouslySetInnerHTML={{ __html: reason.value.Column_paragraph }}
-					></p>
-				</MDBCol>
-			);
-		});
+    let rtn = data.whyColumns.map((reason, i) => {
```
Comment (kleberbaum) [inline src/components/organisms/Sections/Features/index.jsx:17]: Please add a newline.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: public/fav/manifest.json (hunk 13 above)
Comment (Aichnerc) [reply public/fav/manifest.json:2]: Ah yes. This file has been auto generated.

File: src/components/organisms/Sections/Features/index.jsx (hunk 14 above)
Comment (Aichnerc) [reply src/components/organisms/Sections/Features/index.jsx:17]: Prettier does not approve. Wouldn't make sense though. There is a single return in this map function without anything else going on.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**